apps/spruce/cypress/integration/image/build_information.ts

```diff
@@ -6,4 +6,11 @@ describe("build information", () => {
       cy.contains("ubuntu1804-workstation");
     });
   });
+
+  describe("packages", () => {
+    it("should show the corresponding packages", () => {
+      cy.visit("/image/ubuntu2204");
+      cy.dataCy("packages-table-row").should("have.length", 10);
+    });
```

Comment:
Is there anything else we can test here like applying a search, pagination? The reason the distros table test is so short is because it has no search or pagination functionalities